.github/workflows/pr.yaml

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+name: pr
+
+on:
+  push:
+    branches:
+      - "pull-request/[0-9]+"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  pr-builder:
+    needs:
+      - checks
+      - wheel-build
+      - wheel-test-amd64
+      - wheel-test-arm64
+    secrets: inherit
+    uses: rapidsai/shared-action-workflows/.github/workflows/pr-builder.yaml@branch-23.12
+  checks:
+    secrets: inherit
+    uses: rapidsai/shared-action-workflows/.github/workflows/checks.yaml@branch-23.12
+    with:
+      enable_check_generated_files: false
+  wheel-build:
+    runs-on: linux-${{ matrix.arch }}-cpu4
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10"]
+        arch: ["arm64", "amd64"]
+    container:
+      # Use CentOS 7 for an old glibc version, so the wheel has broad manylinux compatibility.
+      image: "rapidsai/ci-wheel:cuda12.0.1-rockylinux8-py${{ matrix.python-version }}"
```

Comment:
The comment says CentOS 7 but the image is Rocky 8. How do we reconcile this? I think we want to build the amd64 wheels on CentOS 7 and arm64 on Rocky 8, to align with our preferred RAPIDS manylinux tags.

Reply:
I originally switched this because on the arm runners, there appeared to be no matching container
```
no matching manifest for linux/arm64/v8 in the manifest list entries
```
whereas there were rockylinux containers available for arm and x86. However I think I can implement what you are saying by separating the build phase into two pieces like the testing phase is. I'll try and update here.

Reply:
this has been done.